t/08-autotest.t

```diff
@@ -486,6 +486,22 @@ subtest 'start_process' => sub {
     stderr_like { $process->{code}->(); } qr/Snapshots are not supported/, 'run_all outputs status on stderr';
 };
 
+subtest 'lua_use' => sub {
+    my $lua_vars = {};
+    $mock_autotest->redefine('lua_set' => sub ($k, $v) { $lua_vars->{$k} = $v; });
+    autotest::_lua_use('testapi');
+    is $lua_vars->{realname}, 'Bernhard M. Wiedemann', 'Check importing strings';
+    is ref($lua_vars->{assert_script_run}), 'CODE', 'Check importing functions';
+
+    use lib './t/data/tests/lib';
+    use testlib;
```

Comment:
These lines are still executed at compile time before the preceding lines of code are executed. Hence I suggest you move them at the top where they belong to avoid giving a false impression of when the lines are executed.